ui/imports/shared/popups/CommunityMembershipSetupDialog.qml

```diff
@@ -84,12 +86,31 @@ StatusStackModal {
     rightButtons: [d.shareButton, finishButton]
 
     finishButton: StatusButton {
-        enabled: {
-            if (root.isInvitationPending || d.accessType !== Constants.communityChatOnRequestAccess)
+        interactive: {
+            if (root.isInvitationPending)
+                return true
+
+            if (root.requirementsCheckPending && !root.joinPermissionsCheckSuccessful)
+                return false
+
+            if (root.joinPermissionsCheckSuccessful || d.accessType !== Constants.communityChatOnRequestAccess)
```

Comment:
@jrainville I believe this is wrong; the `joinPermissionsCheckSuccessful` means that the call itself succeeded, not that we may join CC @MishkaRogachev

Reply:
(same above and below)